gradle-plugin/src/main/kotlin/com/gradle/cucumber/companion/GenerateCucumberSuiteCompanionTask.kt

```diff
@@ -57,7 +63,10 @@ abstract class GenerateCucumberSuiteCompanionTask : DefaultTask() {
         )
         inputChanges.getFileChanges(cucumberFeatureSources).filter { it.file.name.toString().endsWith(".feature") }
             .forEach { change ->
-                val companionFile = CompanionGenerator.resolve(inputDir, outputDir, change.file.toPath())
+                val actual = change.file.toPath()
+                val inputDir = inputDirs.find { actual.startsWith(it) }
+                    ?: throw IllegalArgumentException("File ${change.file} is not in any of the input directories")
```

Comment:
❔ Should we tell users how to fix it? 
```suggestion
                    ?: throw IllegalArgumentException("File ${change.file} must be in one of the 'cucumberFeatureSources' directories to be processed by the plugin")
```